.github/workflows/systest.yml

```diff
@@ -5,7 +5,7 @@ on:
     inputs:
       test_name:
         description: "Test to run"
-        default: "."
+        default: "TestCheckpoint"
```

Comment:
When running the workflow via workflow dispatch, you can specify which tests to run in a text box (a GH Actions button).

Reply:
And if running locally you can provide the `test_name` via command line, Don't forget to revert this before merging 🙂

Reply:
In any case, changing this did not actually have the intended result and I didn't investigate why. I had to skip the tests manually with `t.Skip`

Reply:
It's only used when using `worflow_dispatch`, not `bors try`.